assets/css/styles.css

```diff
@@ -1241,3 +1241,13 @@ a {
     left: 18%;
   }
 }
+
+/*img class is to transform image in bigger when move the cursor on image */
```

Comment:
```suggestion
/* used for zooming in maps in reaching the venue page */
```

Reply:
I removed the css property itself. Since when zoom in bottom and top portion of the image view is cropped or hidden. Vijay also suggested to remove the cosmetic effect in meeting.